[I 2025-06-24 19:40:16,357] A new study created in memory with name: no-name-b55f848b-656c-45dd-904b-786df1b016b5
[W 2025-06-24 19:41:09,832] Trial 0 failed with parameters: {'objective': 'reg:squarederror', 'tree_method': 'hist', 'device': 'gpu', 'random_state': 42, 'n_estimators': 1400, 'learning_rate': 0.0862735828664018, 'max_depth': 6, 'subsample': 0.06868922977663994, 'colsample_bytree': 0.19361118426546192, 'colsample_bylevel': 0.1935967122017216, 'colsample_bynode': 0.13485016730091967, 'lambda': 133.94334706750476, 'alpha': 39.91305878561679, 'gamma': 2.1242177333881367} because of the following error: XGBoostError('[19:41:09] /workspace/src/data/gradient_index.h:100: Check failed: valid: Input data contains `inf` or a value too large, while `missing` is not set to `inf`\nStack trace:\n  [bt] (0) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x2a6ecc) [0x1484e8f6eecc]\n  [bt] (1) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5d5f71) [0x1484e929df71]\n  [bt] (2) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5e9bac) [0x1484e92b1bac]\n  [bt] (3) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5ea949) [0x1484e92b2949]\n  [bt] (4) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x57b541) [0x1484e9243541]\n  [bt] (5) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(XGQuantileDMatrixCreateFromCallback+0x178) [0x1484e8e813c8]\n  [bt] (6) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(+0xa052) [0x148513f7a052]\n  [bt] (7) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(+0x8925) [0x148513f78925]\n  [bt] (8) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(ffi_call+0xde) [0x148513f7906e]\n\n').
Traceback (most recent call last):
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/optuna/study/_optimize.py", line 197, in _run_trial
    value_or_values = func(trial)
                      ^^^^^^^^^^^
  File "/rds/general/user/nz423/home/kaggle/scripts/drw/run_from_here/../../../do_optuna.py", line 122, in objective
    mdl.fit(X_train, y_train)
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 729, in inner_f
    return func(**kwargs)
           ^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/sklearn.py", line 1222, in fit
    train_dmatrix, evals = _wrap_evaluation_matrices(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/sklearn.py", line 628, in _wrap_evaluation_matrices
    train_dmatrix = create_dmatrix(
                    ^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/sklearn.py", line 1137, in _create_dmatrix
    return QuantileDMatrix(
           ^^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 729, in inner_f
    return func(**kwargs)
           ^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 1614, in __init__
    self._init(
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 1680, in _init
    _check_call(ret)
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 310, in _check_call
    raise XGBoostError(py_str(_LIB.XGBGetLastError()))
xgboost.core.XGBoostError: [19:41:09] /workspace/src/data/gradient_index.h:100: Check failed: valid: Input data contains `inf` or a value too large, while `missing` is not set to `inf`
Stack trace:
  [bt] (0) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x2a6ecc) [0x1484e8f6eecc]
  [bt] (1) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5d5f71) [0x1484e929df71]
  [bt] (2) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5e9bac) [0x1484e92b1bac]
  [bt] (3) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5ea949) [0x1484e92b2949]
  [bt] (4) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x57b541) [0x1484e9243541]
  [bt] (5) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(XGQuantileDMatrixCreateFromCallback+0x178) [0x1484e8e813c8]
  [bt] (6) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(+0xa052) [0x148513f7a052]
  [bt] (7) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(+0x8925) [0x148513f78925]
  [bt] (8) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(ffi_call+0xde) [0x148513f7906e]


[W 2025-06-24 19:41:09,944] Trial 0 failed with value None.
Traceback (most recent call last):
  File "/rds/general/user/nz423/home/kaggle/scripts/drw/run_from_here/../../../do_optuna.py", line 214, in <module>
    do_opuna_optimization(
  File "/rds/general/user/nz423/home/kaggle/scripts/drw/run_from_here/../../../do_optuna.py", line 130, in do_opuna_optimization
    study.optimize(objective, n_trials=n_trials)
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/optuna/study/study.py", line 475, in optimize
    _optimize(
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/optuna/study/_optimize.py", line 63, in _optimize
    _optimize_sequential(
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/optuna/study/_optimize.py", line 160, in _optimize_sequential
    frozen_trial = _run_trial(study, func, catch)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/optuna/study/_optimize.py", line 248, in _run_trial
    raise func_err
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/optuna/study/_optimize.py", line 197, in _run_trial
    value_or_values = func(trial)
                      ^^^^^^^^^^^
  File "/rds/general/user/nz423/home/kaggle/scripts/drw/run_from_here/../../../do_optuna.py", line 122, in objective
    mdl.fit(X_train, y_train)
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 729, in inner_f
    return func(**kwargs)
           ^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/sklearn.py", line 1222, in fit
    train_dmatrix, evals = _wrap_evaluation_matrices(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/sklearn.py", line 628, in _wrap_evaluation_matrices
    train_dmatrix = create_dmatrix(
                    ^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/sklearn.py", line 1137, in _create_dmatrix
    return QuantileDMatrix(
           ^^^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 729, in inner_f
    return func(**kwargs)
           ^^^^^^^^^^^^^^
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 1614, in __init__
    self._init(
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 1680, in _init
    _check_call(ret)
  File "/rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/core.py", line 310, in _check_call
    raise XGBoostError(py_str(_LIB.XGBGetLastError()))
xgboost.core.XGBoostError: [19:41:09] /workspace/src/data/gradient_index.h:100: Check failed: valid: Input data contains `inf` or a value too large, while `missing` is not set to `inf`
Stack trace:
  [bt] (0) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x2a6ecc) [0x1484e8f6eecc]
  [bt] (1) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5d5f71) [0x1484e929df71]
  [bt] (2) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5e9bac) [0x1484e92b1bac]
  [bt] (3) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x5ea949) [0x1484e92b2949]
  [bt] (4) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(+0x57b541) [0x1484e9243541]
  [bt] (5) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/site-packages/xgboost/lib/libxgboost.so(XGQuantileDMatrixCreateFromCallback+0x178) [0x1484e8e813c8]
  [bt] (6) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(+0xa052) [0x148513f7a052]
  [bt] (7) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(+0x8925) [0x148513f78925]
  [bt] (8) /rds/general/user/nz423/home/miniforge3/envs/hydraboost/lib/python3.11/lib-dynload/../../libffi.so.8(ffi_call+0xde) [0x148513f7906e]


